### PR TITLE
Implement frontend prototype page

### PR DESCRIPTION
**Pull request by sonowz**
Made prototype login page with Elm.
You can view page with front-make.sh

Closing `implementation` checkbox in  issue #6

### CODE REVIEW

File: frontend/src/elm/Components/TextBox.elm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  , password : Bool
+  }
+
+-- no defaultText
+init : Model
+init =
+  init2 ""
+
+-- has defaultText
+init2 : String -> Model
```
Comment (dj-shin) [inline frontend/src/elm/Components/TextBox.elm]: Just `init2` doesn't explain much about what it does

File: frontend/src/elm/Components/TextBox.elm [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+init2 text =
+  { text = ""
+  , defaultText = text
+  , password = False
+  }
+
+-- password field
+initpw : Model
+initpw =
+  let
```
Comment (LastOne817) [inline frontend/src/elm/Components/TextBox.elm]: I think these are rather unnecessary newlines

File: frontend/src/elm/Pages/Prototype/Main.elm [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        ( newModel, cmd ) =
+          FileSelectBox.update subMsg model.fileSelectBox
+      in
+        ( { model | fileSelectBox = newModel }, Cmd.map FileSelectBoxMsg cmd )
+
+
+-- SUBSCRIPTIONS
+
+subscriptions : Model -> Sub Msg
+subscriptions model =
```
Comment (dj-shin) [inline frontend/src/elm/Pages/Prototype/Main.elm:66]: Unnecessary newline

**Review by dj-shin — CHANGES_REQUESTED** [review]
Use [webpack](https://webpack.js.org/) for build
 - Remove `front-make.sh`
 - Revert `.gitignore`

Minor coding style fix recommended

File: frontend/src/elm/Pages/Prototype/Main.elm (hunk 3 above)
Comment (sonowz) [reply frontend/src/elm/Pages/Prototype/Main.elm:66]: Elm style guide suggests newline in every type declaration.
See "Types" section in : http://elm-lang.org/docs/style-guide

**Comment by sonowz** [comment]
Fixed requests.

File: frontend/src/elm/Components/PasswordBox.elm [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import TextBox exposing (Msg(..))
+
+-- WARNING : This is obsolete code (left for code reference)
+--           Use 'TextBox.initpw' constructor instead
+
+-- INHERITANCE
+-- TODO: is there a better way to inherit?
+type alias Model = TextBox.Model
+init = TextBox.init
+init2 = TextBox.init2
```
Comment (LastOne817) [inline frontend/src/elm/Components/PasswordBox.elm]: This naming should also be changed

File: frontend/src/elm/Pages/Prototype/Main.elm (hunk 3 above)
Comment (dj-shin) [reply frontend/src/elm/Pages/Prototype/Main.elm:66]: Thanks for information 😄

**Review by dj-shin — APPROVED** [review]

**Pull request merged**